### ISSUE TITLE
Use react-native-google-admob - New Actively Maintained Fork - AndroidX support - PRs welcome

### ISSUE THREAD

**Issue by thecodrr** [sbugert/react-native-admob#459]
**Note: This library has been renamed again (for the last time) to `react-native-google-admob`, everyone using this please update accordingly.**

**Note 2: The package name `react-native-google-admob` seems to be occupied on npm but I have sent the author an email asking for ownership. In the meantime, I urge you to use the repo url to install the package.**

Everyone is welcome to try and use this fork I have created. It will be actively maintained and all PRs are welcome.

Current changes:
1. AndroidX support added
2. Upgraded dependency versions in build.gradle
3. Fixed YellowBox warnings

https://github.com/thecodrr/react-native-google-admob/

### npm:
```
npm i --save https://github.com/thecodrr/react-native-google-admob
```
### yarn:
```
yarn add https://github.com/thecodrr/react-native-google-admob
```

**PRs are most welcome! I don't have iOS/macOS so the iOS part of the library can be maintained by someone else. I can give maintainer access to anyone who wants to take that part up.**

**Comment by Albert0405** [comment]
Can you change the lib name to `react-native-admob-updated`?. 
People usually search react-native (insert name of library)
This helps the lib to be more searchable i guess.

**Comment by RohanSensei** [comment]
a star for u

**Comment by koenpunt** [comment]
A PR to fix the issues integrating with new React Native projects is being addressed here: #472
